src/config/routes.rb

```diff
@@ -8,6 +8,7 @@
     resources :saves, only: :index
   end
 
+  resource :relationships, only: %i[create destroy index]
```

Comment:
this should be nested resource for `users`, it's both 
```
resource :followerships, only: %i[create destroy]
resources :followerships, only: %i[index]
``

to have 
```
POST /users/:user_id/followership
DELETE /users/:user_id/followership
GET /users/:user_id/followerships
```